docs/Modules/ecdsa.mdx

```diff
@@ -54,5 +54,8 @@ const module = await ECDSAOwnershipValidationModule.create({
   moduleAddress: DEFAULT_ECDSA_OWNERSHIP_MODULE,
 });
 ```
-
 Once initialized it can be passed to the smart account [create method](/Account/methods/create) to create a smart account.
+
```

Comment:
we can revert this change as well. and in the methods for smart account. there is already an ongoing PR for methods, I can add all these changes to the smart accounts methods there.